cli/azd/pkg/project/project_config.go

```diff
@@ -136,11 +136,28 @@ func ParseProjectConfig(yamlContent string, env *environment.Environment) (*Proj
 		if svc.Module == "" {
 			svc.Module = key
 		}
+
+		if svc.Language == "" || svc.Language == "csharp" || svc.Language == "fsharp" {
+			svc.Language = "dotnet"
+		}
 	}
 
 	return &projectFile, nil
 }
 
+func Initialize(ctx context.Context, p *ProjectConfig) error {
```

Comment:
This can be declared as an instance func on `ProjectConfig`
```suggestion
func (p *ProjectConfig) Initialize(ctx context.Context) error {
```